### PR TITLE
Add CUDA OpenGL interoperability functions

### PR DESCRIPTION
**Pull request by devjeonghwan**
Added cuda opengl headers and info codes for GL interop.
 

```
CUresult cuGLGetDevices ( unsigned int* pCudaDeviceCount, CUdevice* pCudaDevices, unsigned int  cudaDeviceCount, CUGLDeviceList deviceList )
```
> **Gets the CUDA devices associated with the current OpenGL context.**

```
CUresult cuGraphicsGLRegisterBuffer ( CUgraphicsResource* pCudaResource, GLuint buffer, unsigned int  Flags )
```
> **Registers an OpenGL buffer object.**

```
CUresult cuGraphicsGLRegisterImage ( CUgraphicsResource* pCudaResource, GLuint image, GLenum target, unsigned int  Flags )
```
> **Register an OpenGL texture or renderbuffer object.**

```
CUresult cuWGLGetDevice ( CUdevice* pDevice, HGPUNV hGpu )
```
> **Gets the CUDA device associated with hGpu.**

### CODE REVIEW

**Comment by StaticDefault** [comment]
linux-x86_64 build failure

```
Downloading Packages:
http://vault.centos.org/6.10/os/x86_64/Packages/libgfortran-4.4.7-23.el6.x86_64.rpm: [Errno 12] Timeout on http://vault.centos.org/6.10/os/x86_64/Packages/libgfortran-4.4.7-23.el6.x86_64.rpm: (28, 'Operation too slow. Less than 1 bytes/sec transfered the last 30 seconds')
Trying other mirror.
```
i think there is a problem with gnu fortran distribution server(repo)

File: cuda/src/main/java/org/bytedeco/cuda/presets/cudart.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
 @Properties(inherit = javacpp.class, names = {"linux-x86_64", "linux-arm64", "linux-ppc64le", "macosx-x86_64", "windows-x86_64"}, value = {
     @Platform(include = {"<cuda.h>", "<crt/host_defines.h>", "<device_types.h>", "<driver_types.h>", "<surface_types.h>", "<texture_types.h>",
                          "<vector_types.h>", "<builtin_types.h>", "<cuda_runtime_api.h>", "<driver_functions.h>", "<vector_functions.h>",
-                       /*"<cuda_device_runtime_api.h>", <cuda_runtime.h>"*/ "<cuComplex.h>", "<cuda_fp16.h>", "cuda_fp16.hpp", "<library_types.h>"},
+                       /*"<cuda_device_runtime_api.h>", <cuda_runtime.h>"*/ "<cuComplex.h>", "<cuda_fp16.h>", "cuda_fp16.hpp", "<library_types.h>", "<cudaGL.h>"},
```
Comment (saudet) [inline cuda/src/main/java/org/bytedeco/cuda/presets/cudart.java:41]: Let's also add cuda_gl_interop.h?

Comment (StaticDefault) [reply cuda/src/main/java/org/bytedeco/cuda/presets/cudart.java:41]: Ok!

**Comment by StaticDefault** [comment]
I know how to fix this errors

**Comment by StaticDefault** [comment]
nice!

**Pull request merged**